README.md

```diff
@@ -1,2 +1,75 @@
-# accountservice
-![.NET Core push on develop branch](https://github.com/S65-2-project/AccountService/workflows/.NET%20Core%20push%20on%20develop%20branch/badge.svg?branch=develop)
+# AccountService
+This is the accountservice of the Lisk delegate market project. It was made as a groupproject for Fontys. 
+
+It was made with .NET Core 3.1 and will function as an API for the Lisk Delegate Market. To make fully use of the API it is recommened to also look at the other services in the project.
+
+## External sources
+To run this project you will need to run the following services:
+- MongoDB
+- RabbitMQ
+- Sentry
+
+## Configuration
+This is an example for the appsettings.json file that is needed to configure the application. 
+
+```json
+{
+  "AccountDatabaseSettings": {
+    "AccountCollectionName":"",
+    "ConnectionString": "",
+    "DatabaseName": ""
+  },
+  "Logging": {
+    "LogLevel": {
+      "Default": "",
+      "Microsoft": "",
+      "Microsoft.Hosting.Lifetime": ""
+    }
+  },
+  "AppSettings": {
+    "SecretJWT": ""
+  },
+  "MessageQueueSettings":  {
+    "Uri": "",
+    "Exchange": ""
+  },
+  "Sentry": {
+    "Dsn": "",
+    "IncludeRequestPayload": true,
+    "SendDefaultPii": true,
+    "MinimumBreadcrumbLevel": "",
+    "MinimumEventLevel": "",
+    "AttachStackTrace": true,
+    "Debug": true,
+    "DiagnosticsLevel": ""
+  },
+  "AllowedHosts": ""
+}
+```
+
+## Github Actions
+The project runs on GitHub Actions with 3 different configuration.
+
+1. All feature/* branches will are tested and build.
+2. The develop branch is also develired and will be deployed to our development environment. 
```

Comment:
In addition to the steps of step 1 develop branch pushes will also be deliver to dockerhub and deploy to our develop environment.